LICENSE

```diff
@@ -1371,3 +1371,15 @@ The externally maintained libraries used by Node.js are:
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
     THE SOFTWARE.
   """
+
+- hdr_histogram, located at src/histogram, is licensed as follows:
+  """
+    /**
+     * hdr_histogram.h
+     * Written by Michael Barker and released to the public domain,
+     * as explained at http://creativecommons.org/publicdomain/zero/1.0/
+     *
+     * The source for the hdr_histogram utilises a few C99 constructs, specifically
+     * the use of stdint/stdbool and inline variable declaration.
+     */
+  """
```

Comment:
Please add to [`tools/license-builder.sh`](https://github.com/nodejs/node/blob/master/tools/license-builder.sh) otherwise this will be lost the next time the license is regenerated.

Reply:
Yep, that's on my todo list for today. It was late and didn't feel like doing regex munging lol... :-)